wherehows-dao/src/main/java/wherehows/dao/view/DataTypesViewDao.java

```diff
@@ -37,4 +37,13 @@ public DataTypesViewDao(EntityManagerFactory factory) {
   public List<Map<String, Object>> getAllPlatforms() throws Exception {
     throw new UnsupportedOperationException("Operation not implemented");
   }
+
+  /**
+   * Get docs for all field format types
+   */
+  public Map<String, String> getDocsForAllFieldFormats() throws Exception {
```

Comment:
Can remove the mentioning of Docs, just  getAllFieldFormats() should be fine. 
Also, in the java docs, put a few words to explain the Map key and value